config/manager/kustomization.yaml

```diff
@@ -17,5 +17,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: 127.0.0.1:5000/clowder
-  newTag: b7db066
+  newName: quay.io/cloudservices/clowder
+  newTag: 0a44742
```

Comment:
I think we usually omit changes to this file (it's an auto-generated change from the operator-sdk that we don't actually want to push AFAIK). See Blake's new Makefile helper for this here: https://github.com/RedHatInsights/clowder/pull/587

Run a `git checkout origin/master config/manager/kustomization.yaml`